tests/UI/pages/BO/catalog/productsV2/add/index.ts

```diff
@@ -211,7 +211,9 @@ class CreateProduct extends BOBasePage {
       priceTaxExc: await this.getTextContent(page, this.productHeaderTaxExcluded),
       priceTaxIncl: await this.getTextContent(page, this.productHeaderTaxIncluded),
       quantity: await this.getTextContent(page, this.productHeaderQuantity, false),
-      reference: await this.getTextContent(page, this.productHeaderReference('reference'), false),
+      reference: (await page.locator(this.productHeaderReference('reference')).count())
+        ? await this.getTextContent(page, this.productHeaderReference('reference'), false)
+        : '',
```

Comment:
Maybe create specif functions for this one and others, so we could use individually ?